build/Linux+BSD/portable/copy-libs

```diff
@@ -72,7 +74,7 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sensors.so.5
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
-  copyLib libQt5WebChannel.so.5
```

Comment:
Can I move this line to building on Centos6 function?  Debian doesn't seem to have this library.

Reply:
>  windeployqt

Reply:
> windeployqt